.jenkins

```diff
@@ -55,37 +55,6 @@ pipeline {
                         }
                     }
                 }
-                stage('HIP-ROCm-3.5') {
-                    agent {
-                        dockerfile {
-                            filename 'Dockerfile.hipcc'
-                            dir 'scripts/docker'
-                            additionalBuildArgs '--pull --build-arg BASE=rocm/dev-ubuntu-18.04:3.5'
-                            label 'rocm-docker && vega'
-                            args '-v /tmp/ccache.kokkos:/tmp/ccache --device=/dev/kfd --device=/dev/dri --security-opt seccomp=unconfined --group-add video --env HIP_VISIBLE_DEVICES=$HIP_VISIBLE_DEVICES'
-                        }
-                    }
-                    steps {
-                        sh 'ccache --zero-stats'
-                        sh '''rm -rf build && mkdir -p build && cd build && \
-                              cmake \
-                                -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
3.5 was a `Debug` with C++14 and 3.7 `RelWithDebInfo` with C++17

If we are only to test one combination I would go for C++14 and `Debug` but I think we can afford two builds.

Reply:
OK I have added a second build